01-inputs-required.tf

```
@@ -73,6 +73,11 @@ variable "csi_driver_enabled" {
   description = "A toggle to deploy the csi driver as an add-on"
 }
 
+variable "enable_node_os_channel_upgrade" {
```

```suggestion
variable "node_os_channel_upgrade" {
```

I think makes more sense if you'll be using the value passed in to set this, it can still default to false and otherwise use the passed in value

---

Sounds good, will make the changes here